code/game/turfs/open.dm

```diff
@@ -151,24 +173,26 @@
 	if(wet == TURF_WET_ICE && air.temperature > T0C)
 		MakeDry(TURF_WET_ICE)
 		MakeSlippery(TURF_WET_WATER)
-	switch(air.temperature)
-		if(-INFINITY to T0C)
-			if(wet != TURF_WET_ICE && wet)
-				MakeDry(TURF_WET_ICE)
-				MakeSlippery(TURF_WET_ICE)
-		if(T0C to T20C)
-			wet_time = max(0, wet_time-1)
-		if(T20C to T0C + 40)
-			wet_time = max(0, wet_time-2)
-		if(T0C + 40 to T0C + 60)
-			wet_time = max(0, wet_time-3)
-		if(T0C + 60 to T0C + 80)
-			wet_time = max(0, wet_time-5)
-		if(T0C + 80 to T0C + 100)
-			wet_time = max(0, wet_time-10)
-		if(T0C + 100 to INFINITY)
-			wet_time = 0
-
+	if(wet != TURF_WET_PERMAFROST)
+		switch(air.temperature)
+			if(-INFINITY to T0C)
```

Comment:
if byond still memes out with weird fractions (which I'm almost certain it does, considering a few supernerd toxins tricks that still exist), it's likely very possible even without vv